index.js

```diff
@@ -0,0 +1,14 @@
+$(document).ready(function(){
+
+  /*$(window).load(function(){
+     $('#imageID').fadeOut('slow',function(){
+           setTimeout(function(){
+              $(this).fadeOut('slow');*/
+
+              function practice (){
+
+              }
+
+              $("#container").fadeIn(7000);
```

Comment:
This is a pretty entertaining effect. Band sites can definitely get away with slightly less conventional design. I would try shortening it though. There's a rule of thumb in animation on the web that however long you feel it should be - cut it in half, and then cut it in half again. When we work on something so closely it starts to feel faster, but to a new visitor to the site it feels quite long.

Try 2 or 3 seconds and see what you think.